src/Model/Entity/Transcription.php

```diff
@@ -140,4 +154,80 @@ private function getSourceScript($sourceLang) {
         }
         return false;
     }
+
+    protected function _getHtml() {
+        if (!$this->sentence) {
+            return;
+        }
+
+        $lang = $this->sentence['lang'];
+        $text = htmlentities($this->text);
+
+        if ($this->script == 'Hrkt') {
+            $text = $this->rubify($this->text);
+        } elseif ($lang == 'cmn' && $this->script == 'Latn') {
+            $text = $this->numeric2diacritic($text);
+        }
+
+        return $text;
+    }
+
+    protected function _getEditingFormat() {
```

Comment:
Consider renaming "editing format" to "[markup](https://en.wikipedia.org/wiki/Markup_language)".